react-frontend/src/LoginPage.jsx

```diff
@@ -65,11 +65,14 @@ function UserLogin({ setLoginToken }) {
           if (user.password === password) {
             
             // this ensures that the user's data is accesible from other paths 
-            setLoginToken(email);
-            setIsLoggedIn(true);
-            
+            /* eslint no-underscore-dangle: ["error", { "allow": ["_id"] }] */
+            setLoginToken(user._id);
```

Comment:
Great idea changing this to use the id rather than the user's email, not sure how I didn't think of this before. Much more efficient